zebra/kernel_netlink.c

```diff
@@ -1257,6 +1278,32 @@ int netlink_talk(int (*filter)(struct nlmsghdr *, ns_id_t, int startup),
 	return netlink_talk_info(filter, n, &dp_info, startup);
 }
 
+/*
+ * Synchronous version of netlink_talk_info. Converts args to suit the
+ * common version, which is suitable for both sync and async use.
+ */
+int ge_netlink_talk(int (*filter)(struct nlmsghdr *, ns_id_t, int startup),
+		    struct nlmsghdr *n, struct zebra_ns *zns, bool startup)
+{
+	struct zebra_dplane_info dp_info;
```

Comment:
It is not an issue because the functions defined in `kernel_netlink.c` exist only if `HAVE_NETLINK`, as stated in https://github.com/FRRouting/frr/pull/12261#discussion_r1015970668.

If no `HAVE_NETLINK`, `ge_netlink_talk` will not exist and cannot use uninitialized variables :)